package.json

```diff
@@ -23,7 +23,7 @@
   },
   "dependencies": {
     "debug": "2.1.3",
-    "superagent": "1.2.0",
+    "superagent": "^3.8.3",
```

Comment:
Should we pin it to a patch or specific release (like we had previously)?

Reply:
Yeah that's probably better for this...3ee7baa